jmix-flowui/flowui-themes/src/main/resources/META-INF/resources/themes/jmix-lumo/app/main-view.css

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
+nav {
```

Comment:
You must not change style of html element. In case of MainView, corresponding nav element has custom class name: `jmix-main-view-navigation`. Provide styles for it.